### DISCUSSION TITLE
表单组件使用的问题

### DISCUSSION THREAD

**Discussion by AeolusZane** [ant-design/ant-design#34790] [General]
### What problem does this feature solve?

表单组件是否统一都需要实现一套Form.Item透传的属性，这样设计是否会对开发规则限制过多，我如果想在Form.Item内实现一个自己的组件，就需要考虑api。

### What does the proposed API look like?

有没有办法更简单的处理表单项组件，和”表单“解耦，让我在开发表单项组件时只用考虑组件本身，不用考虑它作为一个表单项组件要去实现些什么功能。

**Comment by tushar1998** [comment]
I encountered this issue while using react-hook-form triggering validation occurs only if wrapped inside antd.Form component otherwise it wont get triggered

Solution is using Input status prop and conditionally render warning message and error meesages
